corehq/apps/dhis2/models.py

```diff
@@ -172,5 +197,19 @@ def put(self, path, data, **kwargs):
         return JsonApiRequest.json_or_error(response)
 
 
-# Just for now. Use Documents when we need to persist
-DataValue = namedtuple('DataValue', ('dataElement', 'period', 'orgUnit', 'value'))
+class AsDictMixin(dict):
+    def _asdict(self):
+        # Used by json_serializer for sending to the API. Don't return CouchDB's properties
+        return {k: v for k, v in dict(self).items() if k not in COUCH_KEYS}
+
+
+class DataValue(DocumentSchema, AsDictMixin):
```

Comment:
aren't `DocumentSchema` supposed to be used in another Document's property? `SchemaProperty(DataValue)`